taxcalc/records.py

```diff
@@ -7,7 +7,11 @@
 import numpy as np
 import os.path
 import os
+from numba import vectorize, float64
+from .parameters import Parameters
 from pkg_resources import resource_stream, Requirement
+from .decorators import *
```

Comment:
Now that you are not using decorators from `decorators.py`, this import is also not needed, correct?